--- a/src/Form/ListFormTrait.php
+++ b/src/Form/ListFormTrait.php
@@ -42,9 +42,20 @@
     // Build table.
     $rows = [];
     foreach ($mediaEntities as $media) {
+      $imageUrl = '';
+      if ($media->bundle() === 'image') {
+        $file = \Drupal\media\Entity\Media::load($media->id());
+        $uri = $file->get('field_media_image')->entity->getFileUri();
+        $style = \Drupal\image\Entity\ImageStyle::load('thumbnail');
+        $imageStyleUri = $style->buildUrl($uri);
+        $newUrl = \Drupal\Core\Url::fromUri($imageStyleUri);
+        $imageUrl = $newUrl->toString();
+      }
+
       $rows[$media->id()] = [
         'title' => Markup::create('<strong>' . $media->label() . '</strong>'),
         'id' => Markup::create('<strong>' . $media->id() . '</strong>'),
+        'image' => $imageUrl ? Markup::create('<img src="' . $imageUrl . '" alt="' . $media->label() . '">') : '',
         'type' => Markup::create('<strong>' . $media->bundle() . '</strong>'),
         'link' => Markup::create('<strong><a href="' . Url::fromRoute('entity.media.edit_form',
             [
@@ -82,6 +93,7 @@
     ];
 
     $tableHeader = [
+      'image' => '',
       'title' => $this->t('Title'),
       'id' => $this->t('ID'),
       'type' => $this->t('Type'),